sources/packages/backend/libs/sims-db/src/entities/student-disability-status.type.ts

```diff
@@ -10,3 +10,14 @@ export enum DisabilityStatus {
   PPD = "PPD",
   Declined = "Declined",
 }
+
+/**
+ *  Disability status description.
+ */
+export const DisabilityStatusDescription: Record<DisabilityStatus, string> = {
```

Comment:
Why is this conversion required in API?

For UI purpose we already have a composable 
![image](https://github.com/bcgov/SIMS/assets/54600590/b26ad156-ed62-4320-b025-931228c59458)

Reply:
Yes, ok. I'm going to change to use the UI logic.